Examples/BinaryBH/GNUmakefile

```diff
@@ -34,6 +34,7 @@ src_dirs := $(GRCHOMBO_SOURCE)/utils \
             $(GRCHOMBO_SOURCE)/GRChomboCore  \
             $(GRCHOMBO_SOURCE)/AMRInterpolator  \
             $(GRCHOMBO_SOURCE)/TaggingCriteria  \
+            $(GRCHOMBO_SOURCE)/ApparentHorizonFinder  \
             $(GRCHOMBO_SOURCE)/InitialConditions/BlackHoles \
             $(GRCHOMBO_SOURCE)/BlackHoles
```

Comment:
Can we ifdef the inclusion of the PETSC libs as we do for the two punctures stuff? This will help when we make new examples from these, although I did add it in the wiki instructions, it is not guarantees that people will read them...

Reply:
I'm sorry I think I wasn't very clear here. The inclusion of the ApparentHorizonFinder folder as source should be fine - if users have this code they definitely have that and we shouldn't need to assign it an environmental variable - so no need for an AH_FINDER_SOURCE. I was thinking rather of the $(PETSC_INSTALL)/include directory which we seem to need to add here in addition to allow it to find petsc.h - for me at least I have to add that by hand to the GNUmakefile - do you do it another way?

Reply:
Ah I see, sorry I misunderstood you! I do not add anything additionally to GNUMakefiles. I usually export paths to PETSC_DIR and PETSC_ARCH in my bash_profile, and that's it :)

Reply:
Ah really... hmm, I must be doing something silly, because I do that but still seem to need to add it to the src dirs here...

Reply:
Hmm, maybe it depends on the installation... do you load the petsc module? I have my own version on cosma, do not remember my exact steps but I installed it as a module file, so I am able to 'module load path/to/petsc-library"; perhaps this sets the path correctly.

Reply:
Ah perhaps... in this case we should just revert these files to what you had before, sorry. We can just say in the wiki that if you get errors complaining about not being able to find petsc.h you may need to include the paths manually in your GNUMakefile.

Reply:
Yes, no problem at all, will do that now .

Reply:
I will update the wiki page to reflect this.